src/msg/async/rdma/RDMAConnectedSocketImpl.cc

```diff
@@ -603,7 +603,7 @@ void RDMAConnectedSocketImpl::notify()
   int ret;
 
   ret = write(notify_fd, &i, sizeof(i));
-  assert(ret = sizeof(i));
+  assert(ret == sizeof(i));
```

Comment:
@joscollin And I believe sizeof() check has to be there.
Since sizeof() returns sizeof of type which can be platform dependent. 
We cannot compare return value of write() with an value.